pkg/xray/provider_test.go

```diff
@@ -29,15 +30,25 @@ func TestProvider(t *testing.T) {
 }
 
 func getTestResty(t *testing.T) *resty.Client {
-	if v := os.Getenv("JFROG_URL"); v == "" {
-		t.Error("JFROG_URL must be set for acceptance tests")
+	var ok bool
+	var xrayUrl string
+	if xrayUrl, ok = os.LookupEnv("XRAY_URL"); !ok {
```

Comment:
Added these for now to support both env vars. Need to move `GetEnvVarWithFallback ` from https://github.com/jfrog/terraform-provider-artifactory/blob/master/pkg/acctest/test.go#L75 to the shared module first.